### DISCUSSION TITLE
Adding Additional Network in VM

### DISCUSSION THREAD

**Discussion by jsa4000** [utmapp/UTM#4705] [UTM Help (macOS)] [answered]
Describe the issue

There are two scenarios:

1. Create a Linux VM (ubuntu-22.04.1-arm64) with one interface (Shared Network) . Install the OS. Then shutdown the VM and configure a additional Network, for example Emulated VLAN. Start again the VM. If you list the interfaces there are only one.

2. Create a Linux VM (ubuntu-22.04.1-arm64) with two interfaces (Shared Network and  Emulated VLAN). Install the OS. If you list the interfaces there are two interfaces.

This means you cannot add additional interfaces once the OS has been installed from the first time?

Configuration

    UTM Version: 4.0.9
    macOS Version: 13.0.1
    Mac Chip (Intel, M1, ...): Intel

Crash log
N/A

Debug log
N/A

Upload VM
N/A

**Comment by osy** [comment, marked as answer]
This is an issue with Ubuntu (and maybe other distros too?). They do not automatically configure new network interfaces. You need to manually configure them and update your netplan config (or maybe install some tool that does this for you).

**Reply by jsa4000** [reply]
> This is an issue with Ubuntu (and maybe other distros too?). They do not automatically configure new network interfaces. You need to manually configure them and update your netplan config (or maybe install some tool that does this for you).

Thank you OSY!! 

I have just tests with different Ubuntu versions so it make sense...  I have been used VirtualBox with Intel, but recently I have purchased an M1 Silicon so I have to migrate to UTM/QEMU. Perhaps VirtualBox configure the Netwrok intefaces with every change...
